Find a functionally equivalent code:android.media.MediaRouter.RouteInfo.requestUpdateVolume:COMMENT
Method Modifier: public      
Comment:/**
 * Request an incremental volume update for this route.
 * @param direction Delta to apply to the current volume
 */

@@ -7,12 +7,12 @@
             Log.e(TAG, "Error setting local stream volume", e);
         }
     } else {
-        Log.e(TAG, getClass().getSimpleName() + ".requestChangeVolume(): " + "Non-local volume playback on system route? " + "Could not request volume change.");
+        sStatic.requestUpdateVolume(this, direction);
     }
 }

